lib/generator/files/track_files.rb

```diff
@@ -53,6 +27,16 @@ def exercise_path
         File.join(paths.track, 'exercises', exercise_name)
       end
 
+      # this method contains a LOT of magic text
+      def tests_template_filename
+        File.exist?(track_tests_template_filename) ? track_tests_template_filename :
+          File.join(paths.track, 'lib', 'generator', 'test_template.erb')
+      end
```

Comment:
This is a good place to start, and what I was expecting to see.

Start simple and expand later.